src/app/components/vertical-icons-component.js

```diff
@@ -52,7 +52,14 @@ class VerticalIconComponent {
   listenOnStatus (api) {
     if (!api.events) return
     const fn = (status) => {
-      this.setIconStatus(api.profile.name, status)
+      if (
```

Comment:
why do we need to test the inputs before calling `setIconStatus` ?

Reply:
to limit status types